pkg/controllers/managementapi/usercontrollers/usercontroller.go

```diff
@@ -73,10 +76,15 @@ func Register(ctx context.Context, scaledContext *config.ScaledContext, clusterM
 	}()
 }
 
+type controllerStarter interface {
+	Start(ctx context.Context, cluster *v3.Cluster, clusterOwner bool) error
+	Stop(cluster *v3.Cluster)
+}
+
 type userControllersController struct {
 	sync.Mutex
 	clustered     bool
-	manager       *clustermanager.Manager
+	manager       controllerStarter
```

Comment:
Nit: This now has a type of controllerStarter but you are still calling the variable by the name of the struct which implements the interface. This should probably be a `starter`, and  comment should be left above the interface to note:
- What the interface is for
- That clustermanager.Manager implements the interface (optional - this might be nice to have but could technically be a bad practice).